src/python_testing/TC_DEM_Utils.py

```diff
@@ -0,0 +1,220 @@
+#
```

Comment:
what are `Utils` in the name of this test?

Could we name things something else? Utils/support/helper/core are filler words only.

Reply:
I can rename this after the class defined in this module - dembasetesthelper.py (which makes more sense anyway) if that's ok?

Reply:
I am complaining about both the file name and the class name. Helper is a filler word. Base partially so. What do these do? Are they `DEMTestCommands` or somethine else?

Why did we create this separate module instead of inlining some things?

Reply:
I agree it is badly name module. The class defined in it is a base class for all DEM tests. They inherit a load of common methods from this class which is called DEMBaseTestHelper.

Maybe a better name for the class is DEMTestBase and the module could be renamed likewise?

Reply:
Have renamed file to DEMTestBase + renamed the class within that module DEMTestBase